### PR TITLE
Pi hole controller

### PR DESCRIPTION
**Pull request by nate-ubiquisoft**
Fixes #[issue number] .

Changes proposed in this pull request:

- Adding single script `pihole.sh` to symlink to /usr/local/bin from /opt/pihole
 - `pihole.sh` will call other scripts such as `whitelist.sh` and `blacklist.sh`

@pihole/gravity

### CODE REVIEW

**Comment by Mcat12** [comment]
Shouldn't you should remove the `.sh` from the file name, so you can do `pihole ...`?

**Comment by nate-ubiquisoft** [comment]
@Mcat12 Yes I was planning on doing that I just havent gotten a chance to work on the controller since my initial commits.

**Comment by nate-ubiquisoft** [comment]
@pi-hole/gravity Note that this is branched from my development branch in PR #411

File: pihole.sh [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+
+# Must be root to use this tool
+if [[ $EUID -eq 0 ]];then
+	echo "::: You are root."
+else
+	echo "::: Sudo will be used for this tool."
+  # Check if it is actually installed
+  # If it isn't, exit because the unnstall cannot complete
```
Comment (Mcat12) [inline pihole.sh]: Copy error. this isn't uninstall.

File: pihole.sh [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    echo "::: Options:"
+    echo ":::  -w, --whitelist 			Whitelist domains"
+    echo ":::  -b, --blacklist 			Blacklist domains"
+    echo ":::  -d, --debug 				Start a debugging session if having trouble"
+    echo ":::  -f, --flush 				Flush the pihole.log file"
+    echo ":::  -u, --updateDashboard 	Update the web dashboard manually"
+    echo ":::  -g, --updateGravity		Update the list of ad-serving domains"
+   	echo ":::  -s, --setupLCD 			Automatically configures the Pi to use the 2.8 LCD screen to display stats on it"
+	echo ":::  -c, --chronometer 		Calculates stats and displays to an LCD"
+	echo ":::  -h, --help 				Show this help dialog"
```
Comment (Mcat12) [inline pihole.sh]: Maybe instead of writing them as flags, write them like `pihole whitelist ...` or `pihole debug`

Comment (nate-ubiquisoft) [inline pihole.sh]: I was adding the flags for shorthand notation but I can do that. Are you okay with me adding both? I can just remove the `--` from the fully qualified command, i.e. `-w, whitelist`

Comment (Mcat12) [inline pihole.sh]: That's fine. :+1:

**Comment by PromoFaux** [comment]
image
Another case of missing `cat`s? :)

**Pull request merged**